### PR TITLE
Using 'nameof' operator instead of magic string

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: test/Microsoft.AspNet.SignalR.ServiceBus.Tests/ServiceBusConnectionFacts.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public void RetryRetriesActionOnStandardException()
             {
                 ++called;
                 if (called < 2)
-                    throw new Exception("Error");
```
Comment (moozzyk) [inline test/Microsoft.AspNet.SignalR.ServiceBus.Tests/ServiceBusConnectionFacts.cs:40]: This is wrong. This `Exception` ctor takes a message as the parameter and not parameter name. This change does not add any value and actually makes the code more confusing. Same applies to changes below. 

(From what I see a good change in this file would be to remove `ref` from the `ref int called` parameter on the `RetryHelper` method - it does not seem to be changed inside the method and therefore does not have to be `ref` -  the tests still must build and pass after the change).

File: src/Microsoft.AspNet.SignalR.ServiceBus/ServiceBusScaleoutConfiguration.cs [hunk 2]
```diff
@@ -25,19 +26,9 @@ public ServiceBusScaleoutConfiguration()
             OperationTimeout = null;
         }
 
-        public ServiceBusScaleoutConfiguration(string connectionString, string topicPrefix)
+        public ServiceBusScaleoutConfiguration([NotNull] string connectionString, [NotNull] string topicPrefix)
```
Comment (Eilon) [inline src/Microsoft.AspNet.SignalR.ServiceBus/ServiceBusScaleoutConfiguration.cs]: Empty string != null...

**Comment by hishamco** [comment]
@Eilon I update the PR to use nameof and left the NotNull while I have seen @pranavkm applying a script to throw ArgumentNullException instead

**Comment by hishamco** [comment]
@Eilon can you have a look and merge if it's fine

**Comment by hishamco** [comment]
@Eilon can you have a look and merge if it's fine

**Comment by pranavkm** [comment]
We're no longer building this repo.

**Pull request closed without merging**